src/log.rs

```diff
@@ -15,14 +17,14 @@ pub enum LogLevel {
 }
 
 impl LogLevel {
-  pub fn color(&self) -> u8 {
+  pub fn fg_color(&self) -> XtermColors {
     match self {
-      Self::Trace => 81,
-      Self::Debug => 202,
-      Self::Info => 10,
-      Self::Warn => 11,
-      Self::Error => 9,
-      Self::Fatal => 9,
+      Self::Trace => XtermColors::MalibuBlue,
+      Self::Debug => XtermColors::BlazeOrange,
+      Self::Info => XtermColors::UserGreen,
+      Self::Warn => XtermColors::UserYellow,
+      Self::Error => XtermColors::UserRed,
+      Self::Fatal => XtermColors::UserRed,
```

Comment:
Nice! I chose to do color codes in the interest of minimalism, but I think this crate looks cool (and not too big)!

Reply:
I had the same thought, this crate is pretty good and unintrusive. I used to use `colored` but this one is much better. I think using the color codes is not really maintainable and I'm glad you don't mind the additional dependency.

Reply:
Yeah I didn't know this existed! I don't remember the ones I looked at when I was building this, but they all looked far too bulky, so I sorta just did the absolute simplest thing. I agree that it probably wasn't maintainable.